### DISCUSSION TITLE
Triggering signup from manually creates duplicate entry with Auth provider

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-discussions#56] [Extension Development QnA] [answered]
Hello everyone, i was trying to implement the custom authentication provider functionalities from this blog post https://www.eliostruyf.com/create-authentication-provider-visual-studio-code/ (which works perfectly fine). I would like to show a notification to the user using the showInformationMessage feature and trigger the sign up manually. Though it ends up creating two sessions, one which was already registered (not logged in state) and one triggered manually (logged in state). Any suggestions?

**Comment by isidorn** [comment, marked as answer]
Are you still facing this? Ideally only one session should be created. So it might be you are doing something wrong. 
Did you read the Authentication API in detail? https://code.visualstudio.com/api/references/vscode-api#authentication

**Reply by ghost** [reply]
It's possible that it's something that I have done, i still haven't figured it out. Will check the documentation more, thanks

**Reply by isidorn** [reply]
If you can provide some easy reproducable steps then we can ask somebody from the dev team.
